plugin/inlay_hint.py

```diff
@@ -162,10 +174,18 @@ def format_inlay_hint_label(inlay_hint: InlayHint, session: Session, phantom_uui
                 }
             })
             value += f'<a href="{inlay_hint_click_command}">'
-        value += html.escape(label_part['value'])
+        raw_label = label_part['value']
+        truncated = len(raw_label) > remaining_truncate_limit
+        truncated_label = raw_label[:remaining_truncate_limit] + '…' if truncated else raw_label
+        remaining_truncate_limit -= len(raw_label)
+        value += html.escape(truncated_label)
         if has_command:
             value += "</a>"
         # InlayHintLabelPart.location is not supported
         instruction_text = '\nDouble-click to execute' if has_command else ""
         result += f"<span title=\"{(tooltip + instruction_text).strip()}\">{value}</span>"
+    if truncated:
+        tooltip_label = "".join(label_part['value'] for label_part in label)
+        truncation_tooltip = html.escape('\n' + tooltip_label)
```

Comment:
It doesn't need to start with a newline here.

Reply:
Done in [bc41741](https://github.com/sublimelsp/LSP/pull/2514/commits/bc41741055fed8ccac2f7b91a2372baea218e214)